static/js/htmx-integration.ts

```diff
@@ -38,6 +39,13 @@ htmx.defineExtension('disable-element', {
  */
 htmx.onLoad((content) => {
     initializeHighlightedCodeBlocks(content);
+    var sortable : HTMLElement =  content.querySelector('.sortable') as HTMLElement;
```

Comment:
In the [example here](https://htmx.org/examples/sortable/) they're also taking care of the possibility of there being more than one `.sortable` element. Something we'd want to do as well, just to be sure?

Reply:
Ok! Will do!